docs/language/expressions.md

```diff
@@ -342,10 +344,6 @@ yield f"pi is approximately {numerator / denominator}"
 If any expression results in an error, then the value of the f-string is the
 first error encountered in left-to-right order.
 
-> TBD: we could improve an error result here by creating a structured error
-> containing the string template text along with a list of values/errors of
-> the expressions.
-
```

Comment:
I thought this text was making a good point, but we generally track enhancements to-do items in issues so I opened #5902 and am proposing deleting the text here. If there's consensus that this is important enough to still mention in the doc I'd be open to morphing this text into something that includes a link to the open issue, but my prior attempts to include issue links in docs text haven't always been well-received in the past so I assume folks would prefer to drop it.